.infrastructure/nodeport.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: todoapp-nodeport
+  namespace: todoapp
+spec:
+  type: NodePort
+  selector:
+    app: todolist
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 8080
+      nodePort: 30007
```

Comment:
![image](https://github.com/user-attachments/assets/67ca2ef9-fe27-45c4-adaf-97869080153f)

Add Eof(End-of-file), need to add a blank line at the end

Reply:
fixed